account_payment_method_or_mode_fs_storage/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.company"
+
+    fs_storage_source_payment = fields.Selection(
```

Comment:
IMO this is strange. It depends on where the storage is filled.
That means `mode` and `method` can not be mixed?